src/AspNet.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.cs

```diff
@@ -647,17 +647,17 @@ internal class OpenIdConnectServerHandler : AuthenticationHandler<OpenIdConnectS
                 await Options.Cache.RemoveAsync(identifier);
             }
 
-            var notification = new AuthorizationEndpointResponseNotification(Context, Options, request, response);
-            await Options.Provider.AuthorizationEndpointResponse(notification);
+            var authorizationContext = new AuthorizationEndpointResponseContext(Context, Options, request, response);
+            await Options.Events.AuthorizationEndpointResponse(authorizationContext);
 
-            if (notification.HandledResponse) {
+            if (authorizationContext.HandledResponse) {
                 return;
             }
 
             await ApplyAuthorizationResponseAsync(request, response);
         }
 
-        protected override async Task HandleSignOutAsync(SignOutContext context) {
+        protected override async Task HandleSignOutAsync(SignOutContext signOutContext) {
```

Comment:
Hum. If this collides with another context/notification, keep using notification.